config/evaluate.yaml

```diff
@@ -1,16 +1,34 @@
-run:
-    output_dir: evaluations
-
 gold_files:
-    source_tags: data/WMT19/sentence_level/en_de.nmt/dev.source_tags
-    target_tags: data/WMT19/sentence_level/en_de.nmt/dev.tags
-    sentence_scores: data/WMT19/sentence_level/en_de.nmt/dev.hter
-
-predicted_files:
-    source_tags: source_tags
-    target_tags: target_tags
-    gap_tags: gap_tags
-    sentence_scores: sentence_scores
+    source_tags: data/WMT20/sentence_level/en_de.nmt/dev.source_tags
+    target_tags: data/WMT20/sentence_level/en_de.nmt/dev.tags
+    sentence_scores: data/WMT20/sentence_level/en_de.nmt/dev.hter
 
+# Two configuration options:
```

Comment:
All of these comments should live in our docs somewhere visible! This is very good info.

Reply:
The docs don't really have much info on the details of the evaluation pipeline, but it does show this as an example configuration.

So it will be read :)